packages/web/src/common/store/search-bar/sagas.ts

```diff
@@ -97,3 +97,4 @@ export default function sagas() {
   const sagas = [watchSearch]
   return NATIVE_MOBILE ? sagas.concat(mobileSagas()) : sagas
 }
+// can go in common - mobile sagas should be in mobile
```

Comment:
It looks like you moved the mobileSagas into common, this won't really work once the webview is gone because it does message passing from the webview to the native app. So we need to reimplement that logic inside the native sagas, maybe you are planning to tackle in the next pr?

Reply:
yeah i was going to tackle in the next PR! i think we can probably get rid of all the mobile sagas.